src/effects/EffectManager.cpp

```diff
@@ -111,6 +111,9 @@ TranslatableString EffectManager::GetCommandName(const PluginID & ID)
 
 TranslatableString EffectManager::GetEffectFamilyName(const PluginID & ID)
 {
+   if(auto description = PluginManager::Get().GetPlugin(ID))
+      return TranslatableString { description->GetEffectFamily(), {} };
+
```

Comment:
My recent changes in EffectManager may make the benefits of these changes less but still not zero.  After my changes, instance initialization is avoided, but still not the plugin initialization.